app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -96,14 +122,43 @@ class PoliciesActivityTest {
   @get:Rule
   val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
 
+  @Inject
+  @field:DefaultResourceBucketName
+  lateinit var resourceBucketName: String
+
+  @Inject
+  lateinit var htmlParserFactory: HtmlParser.Factory
+
+  @Inject
+  lateinit var appLanguageLocaleHandler: AppLanguageLocaleHandler
+
   @Inject
   lateinit var context: Context
+  @Rule
+  @JvmField
+  val mockitoRule: MockitoRule = MockitoJUnit.rule()
+
+  @Mock
+  lateinit var mockPolicyOppiaTagActionListener: HtmlParser.PolicyOppiaTagActionListener
+
+  @Captor
+  lateinit var policyTypeCaptor: ArgumentCaptor<PolicyType>
+
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
 
   @Before
   fun setUp() {
+    Intents.init()
     setUpTestApplicationComponent()
+    testCoroutineDispatchers.registerIdlingResource()
   }
 
+  @After
+  fun tearDown() {
+    Intents.release()
+    testCoroutineDispatchers.unregisterIdlingResource()
+  }
```

Comment:
Nit: add newline after this bracket for consistency.

Reply:
Done.